### ISSUE TITLE
split-delivery

### ISSUE THREAD

**Issue by yr18** [mveres01/pytorch-drl4vrp#5]
Hi,
In the paper, the author mentioned if the whole delivery constraint is relaxed the result will get better.

However if I omit the constraint( demands.lt(load) the result is always worse. even if I try to retrain the model.

In the paper even for the greedy, split delivery is better than whole delivery. 

Do you know what may cause the difference here? 

Thank you very much for your help!

**Comment by mveres01** [comment]
Hi!

You're on the right line for split delivery; on [line 80](https://github.com/mveres01/pytorch-drl4vrp/blob/master/tasks/vrp.py#L80), make sure you comment out only the `demands.lt()` portion and not the whole line, otherwise the mask just won't work

I'd take a look at the hyperparameters a little more first. If you consider split delivery in the context of _purely_ learning to route, it feels like a bit of a specialized behaviour in the sense that the salesman is now given the choice of ignoring the city unless it can meet demand, or taking a detour to fulfill part of it. Try either increasing the batch size and increasing LR, or decreasing batch size and decreasing LR. I've had some luck with e.g.:  

`python trainer.py --task=vrp --nodes=10 --batch_size=512 --actor_lr=1e-3 --critic_lr=1e-3`

**Edit**: fixed broken link

**Comment by yr18** [comment]
My case is a VRP problem. I only comment out *demands.lt() part.  I only increased the batch_size before, however it gave me worse results.  I will try your method.

Thank you so much for your reply!
